Translations/Web/lang_ru.php

```diff
@@ -4,7 +4,7 @@
 //
 // Language: Russian
 $content_language_locale = "ru";
-// Translated by NickKaramoff (nick@karamoff.ru)
+// Translated by Nick Karamoff (nick@karamoff.ru)
```

Comment:
This one is from the author so better leave it as he likes :)

https://github.com/aonez/Keka/pull/312/files#diff-f2232528f4d142de92ba0393e18d6ccfR6

Reply:
> This one is from the author so better leave it as he likes :)

Here is an example of the fact that the author himself is not yet sure how to write his first and last name together or separately :)
**Translators_html**
<img width="693" alt="Translators_html" src="https://user-images.githubusercontent.com/62497891/91630637-31237500-e9d3-11ea-8f7d-36373ddb6a11.png">